inlong-sdk/sort-sdk/src/main/java/org/apache/inlong/sdk/sort/impl/SortClientImpl.java

```diff
@@ -39,7 +40,7 @@ public class SortClientImpl extends SortClient {
 
     private final ClientContext context;
 
-    private final InLongTopicManager inLongTopicManager;
+    private final InlongTopicManager inLongTopicManager;
```

Comment:
`inLongTopicManager` -> `inlongTopicManager`.

Reply:
there are too many naming problem, I will make a new issue to fix these.